tests/test_file.py

```diff
@@ -31,7 +31,7 @@
 from .http_mock import HttpMockSequence
 import six
 
-from oauth2client import file
```

Comment:
Mine, it's been bugging me for a while. I personally think if we're going to break everything, we might as well not provide proxies for the old ones. That's @nathanielmanistaatgoogle and your call though.